de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopWindowManager.java

```diff
@@ -5,6 +5,7 @@
 import org.iplantc.de.commons.client.ErrorHandler;
 import org.iplantc.de.commons.client.views.window.configs.ConfigFactory;
 import org.iplantc.de.commons.client.views.window.configs.WindowConfig;
+import org.iplantc.de.desktop.client.views.windows.DEAppsWindow;
```

Comment:
This looks like a stray import that's not actually used in the code.

We can setup your IDE with some in-house standard code-formatting and import cleanups.